pages/docs/reference/compiler-plugins.md

```diff
@@ -117,9 +117,9 @@ plugins {
 The Maven example is similar to the one above.
 
 The plugin specifies the following annotations: 
-[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html), 
-[`@Async`](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/scheduling/annotation/Async.html), 
-[`@Transactional`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html), 
+[`@Component`](http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/stereotype/Component.html)
```

Comment:
I mean, it should be a comma-separated list, as it was originally. It is not vertical, and without commas it looks mis-formatted:

![image](https://user-images.githubusercontent.com/1888526/30456639-277a1c6c-99ad-11e7-9b1f-47959e1505f0.png)